docs/content/toc.md

```diff
@@ -86,8 +86,7 @@
 ## Development
   * [Overview](../development/overview.html)
   * [Libraries](../development/libraries.html)  
-  * [Extending Druid](../development/modules.html)
```

Comment:
I think after this nothing will link to modules.html, is that intentional? Does that doc provide any useful content?